content/blog/announcing-hpe-swarm-learning-2-0-0.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Announcing HPE Swarm Learning 2.0.0
+date: 2023-05-11T04:47:46.996Z
```

Comment:
You may want to change the date to June 1st.

Reply:
Changed to latest.